pkg/provisioner/demo/demo.go

```diff
@@ -78,17 +78,17 @@ func (m *demoProvisioner) HasProvisioningCapacity() (result bool, err error) {
 
 // ValidateManagementAccess tests the connection information for the
 // host to verify that the location and credentials work.
-func (p *demoProvisioner) ValidateManagementAccess(credentialsChanged, force bool) (result provisioner.Result, provID string, err error) {
-	p.log.Info("testing management access")
+func (m *demoProvisioner) ValidateManagementAccess(credentialsChanged, force bool) (result provisioner.Result, provID string, err error) {
```

Comment:
This is a lot of code churn to fix a linter issue. What about changing `m` on line 75 to `p`?